cookbooks/aws-parallelcluster-slurm/libraries/dynamo.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Copyright 2024 Amazon.com, Inc. and its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+def save_cluster_config_version_to_dynamodb
```

Comment:
what about: `store_instance_config_version` or `save_instance_config_version_to_dynamodb`?

I'd like to have instance in the name to clarify that we're saving this "mapping" and not just the config version.

"cluster" might be omitted

Reply:
Sure, I like save_instance_config_version_to_dynamodb